From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Noah Ross <noahwross@gmail.com>
Date: Wed, 19 Apr 2023 10:21:37 -0400
Subject: [PATCH] Added random KB from old versions


diff --git a/src/main/java/net/minecraft/world/level/Explosion.java b/src/main/java/net/minecraft/world/level/Explosion.java
index 59837144c2c0460aca6e8c349eb3d6528111d1dc..5e96a3c8b1ecd4bc8c2c7dd8741fca979dbc0779 100644
--- a/src/main/java/net/minecraft/world/level/Explosion.java
+++ b/src/main/java/net/minecraft/world/level/Explosion.java
@@ -272,7 +272,7 @@ public class Explosion {
                         if (entity instanceof LivingEntity) {
                             LivingEntity entityliving = (LivingEntity) entity;
 
-                            d14 = entity instanceof Player && level.paperConfig().environment.disableExplosionKnockback ? 0 : ProtectionEnchantment.getExplosionKnockbackAfterDampener(entityliving, d13); // Paper - disable explosion knockback
+                            d14 = entity instanceof Player && level.paperConfig().environment.disableExplosionKnockback ? 0 : d13; // Paper - disable explosion knockback //SuperV reverted to pre 1.19.4 netherite KB
                         } else {
                             d14 = d13;
                         }
